src/App/Pages/Lock/LockFingerprintPage.cs

```diff
@@ -32,9 +33,23 @@ public LockFingerprintPage(bool checkFingerprintImmediately)
 
         public void Init()
         {
+            var image_helper = Helpers.OnPlatform(
```

Comment:
This variable should use `camelCase` and also be named something more appropriate, such as `biometricIcon`.